### PR TITLE
44 search and styling

### LINKED ISSUE #44: partners page
### Features to Add
- :heavy_check_mark: display all partners, along with connected ministers
- :heavy_check_mark: pull partners list from state
- :heavy_check_mark: will probably need to create a view table on db to get the connected ministers for each partner before sending initial request for partners
- :heavy_check_mark: add edit and delete functionality
- :heavy_check_mark: better formatting of address
- :heavy_check_mark: loading animations on edit and delete
- :heavy_check_mark: success/error alerts for edit and delete
- :heavy_check_mark: confirmation check before deleting
- :heavy_check_mark: paginate partners list
  - make sure each column is sorting in a way that makes sense - connected ministers should sort by number of connected ministers probably or not have the option to sort this column, and it needs to work on mobile and desktop
- :heavy_check_mark: sort partners list by names in alphabetical order before creating search index
- :heavy_check_mark: change birthdate to be two selection inputs?
- add form validation for edit
- have search only show exact matches based on names, email, minister names
- add functionality to set/change/remove spouse via partner edit form
  - :heavy_check_mark: add X button to remove existing spouse
  - allow arrow keys to move through searchList results
  - stop adding spouse name via postgres view table, just use id to get name from redux store
  - remove `is_married` column on partners, just check if `spouse_id` exists
  - use a merge table to connect spouses
- look into using [supabase realtime database subscriptions](https://supabase.com/docs/reference/javascript/subscribe)?

### Bugs
- :heavy_check_mark: may not be autofocusing when adding a name for partner
- :heavy_check_mark: when searching it's not automatically sorting by search result score

### PR DESCRIPTION
**Pull request by matthew-r-clark**
### Note: hover state over Phillis Hrihorovich.
image

### CODE REVIEW

File: src/pages/partners.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    direction="row"
+    justifyContent="space-evenly"
+    alignItems="center"
+    style={{
+      padding: '3px 0',
+      borderRadius: 5,
+    }}
+    title={generateAkaString(partner)}
+    onClick={() => {
+      alert(JSON.stringify(partner, null, 2));
```
Comment (matthew-r-clark) [inline src/pages/partners.js:57]: This will be replaced by a modal or "card" component in a later PR.

**Review by Jon-D-Smith — APPROVED** [review]
Looks good! Thanks for all the hard work!

File: package.json [hunk 2]
```diff
@@ -6,6 +6,7 @@
     "dev": "next dev",
     "debug": "NODE_OPTIONS='--inspect' next dev",
     "build": "next build",
+    "lint": "npx eslint . --ext .js,.jsx",
```
Comment (MitchellPavel) [inline package.json:9]: why not use [next lint](https://nextjs.org/docs/basic-features/eslint)?

**Review by MitchellPavel — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (matthew-r-clark) [reply package.json:9]: I was unaware of that, but I'll check it out.

Comment (matthew-r-clark) [reply package.json:9]: But what I did was just use the same command we're using in the automated CI check.

**Pull request merged**